src/print-1-10-growing.py

```diff
@@ -1,2 +1,7 @@
 
 # Print the numbers described in the exercise
+x = ''
+for number in range (1,11,1):
+    x += '{}'.format(number)
```

Comment:
With more recent versions of Python, you can now do this with `f"{number}"`. Putting an `f` in front of a string means that you can format it by just putting expressions in curly brackets inside it. There is nothing wrong with using `"...".format(...)`, but the other variant is sometimes easier to read. Here, though, you are just translating `number` into a string, and you can do that with `str(number)`.